examples/ipv6/README.md

```diff
@@ -0,0 +1,111 @@
+# AKS Engine - IPv6 clusters
+
+## Overview
+
+AKS Engine enables you to create IPv6 Kubernetes clusters on Microsoft Azure.
+
+- IPv6 support is available for Kubernetes version 1.18.0 and later on Azure.
+- IPv6 support is in beta as of Kubernetes version 1.18 in Kubernetes community.
+
+In order to create IPv6 enabled Azure virtual networks and use standard loadbalancer with IPv6 you must first configure your subscription [as follows](https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-ipv4-ipv6-dual-stack-cli#prerequisites).
```

Comment:
there is no #prerequisites paragraph on that page

Reply:
also nit: `as follows` implies it's described below, should be `as described [here](link)`

Reply:
The feature is now GA, so they have removed the `prerequisites`. I'll update the doc to reflect that change.